sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/internal/aggregator/DoubleExponentialHistogramBucketsTest.java

```diff
@@ -110,6 +110,6 @@ void testToString(ExponentialBucketStrategy buckets) {
     DoubleExponentialHistogramBuckets b = buckets.newBuckets();
     b.record(1);
     assertThat(b.toString())
-        .isEqualTo("DoubleExponentialHistogramBuckets{scale: 20, offset: 0, counts: {0=1} }");
+        .isEqualTo("DoubleExponentialHistogramBuckets{scale: 20, offset: -1, counts: {-1=1} }");
```

Comment:
The fact that only the offset here is changing by one gives me confidence that the PR is solid.